02-peer-protocol.md

```diff
@@ -545,6 +545,42 @@ considered to be the channel funder (the sender of `open_channel`).
 The quiescence effect is exactly the same as if one had replied to the
 other.
 
+### `go_on`
+
+1. type: TODO (`go_on`)
+2. data:
+    * [`channel_id`:`channel_id`]
+    * **NOTE FOR REVIEWERS** do we need to include initiator here? I think not.
```

Comment:
I don't think so either, since the only valid flow is:

1. the quiescence initiator sends `go_on`
2. the other node responds with `go_on`

Any other flow would be an error, that should trigger a disconnection.